index.html

```diff
@@ -0,0 +1,226 @@
+            disabled>
+            Submit
+          </button>
+        </div>
+      </form>
+    </div>
+    <hr />
+    <div class="content">
+      <h2 class="thoughts">Thoughts</h2>
+      <hr />
+      <p class="oc">
+        Online classes are really convenient. You don't have to wake up early or
+        rush through your morning routine to make it to school on time. You can
+        attend class from home or anywhere with internet, which saves time and
+        removes the stress of commuting. But in face-to-face classes, like when
+        learning programming, I find it easier to focus and learn. Direct
+        interaction with teachers and classmates helps clear up questions
+        quickly and lets us work together on problems. Plus, being in a
+        structured classroom helps me stay focused without distractions. In
+        online classes, distractions like chores and noisy neighbors can make it
+        hard to concentrate. Many students also seem to be less engaged during
+        online classes, doing other activities instead of paying attention. It
+        can sometimes feel like background noise rather than effective learning.
+      </p>
+    </div>
+    <hr />
+    <div class="content">
+      <h2 class="gallery">Gallery</h2>
+      <hr />
+      <div class="polaroid">
+        <img
+          src="https://i.pinimg.com/564x/59/36/72/593672472ecdecbf430b1a91be90d228.jpg"
+          alt="Kuromi"/>
+        <div class="container">
+          <p>Kuromi</p>
+        </div>
+      </div>
+      <div class="polaroid">
+        <img
+          src="https://i.pinimg.com/564x/c5/0a/df/c50adf2760501c21f933074159208088.jpg"
+          alt="Melody"/>
+        <div class="container">
+          <p>Melody</p>
+        </div>
+      </div>
+      <div class="polaroid">
+        <img
+          src="https://i.pinimg.com/564x/f8/1c/62/f81c62405e1825eeb3ef85a6164bf2a6.jpg"
+          alt="Cinnamoroll"/>
+        <div class="container">
+          <p>Cinnamoroll</p>
+        </div>
+      </div>
+      <div class="polaroid">
+        <img
+          src="https://i.pinimg.com/564x/9f/c3/1d/9fc31d6dd8b381178cc5a0b42360faa4.jpg"
+          alt="Keroppi"/>
+        <div class="container">
+          <p>Keroppi</p>
+        </div>
+      </div>
+      <div class="polaroid">
+        <img
+          src="https://i.pinimg.com/564x/74/99/cc/7499cc5ef38c72247042323cc72e6b13.jpg"
+          alt="Hello Kitty"/>
+        <div class="container">
+          <p>Hello Kitty</p>
+        </div>
+      </div>
+    </div>
+    <hr />
+    <div class="details">
+      <h2 class="contacts">Contact Me</h2>
+      <hr />
+      <form id="commentForm">
+        <div class="email">
+          <label for="email">Email:</label>
+          <input type="email" id="email" name="email" required />
+        </div>
+        <div class="fullname">
+          <label for="fullname">Full Name:</label>
+          <input type="text" id="fullname" name="fullname" required />
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. 
See the reference for clarifications. 
```suggestion
          <input type="text" id="full_name" name="fullname" required />
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5